engine/consensus/matching/engine.go

```diff
@@ -674,18 +674,7 @@ func (e *Engine) matchChunk(incorporatedResult *flow.IncorporatedResult, block *
 		validApprovals++
 	}
 
-	// skip counting approvals. We don't put this earlier in the function
-	// because we still want to test that the above code doesn't panic.
-	// TODO: this is only here temporarily to ease the migration to new chunk
-	// based sealing.
-	if !e.requireApprovals {
-		return true, nil
-	}
-
-	// TODO:
-	//   * This is the happy path (requires just one approval per chunk).
-	//   * Full protocol should be +2/3 of all currently staked verifiers.
-	return validApprovals > 0, nil
+	return validApprovals >= e.requiredChunkApprovals, nil
```

Comment:
@AlexHentschel I have used `requiredChunkApprovals` here, currently we can control chunk matching and seal verification with one flag. I don't think that we need the old check `e.requiredChunkApprovals > 0` but need to be sure.

Reply:
Thanks. Your revision is good. 👍